contracts/delegation/Delegation.sol

```diff
@@ -194,6 +201,14 @@ contract Delegation is IDelegation, UUPSUpgradeable, Access, DelegationStorageUt
             ISymbioticNetworkMiddleware($.agentData[_agent].network).slashableCollateral(_agent, _slashTimestamp);
     }
 
+    /// @inheritdoc IDelegation
+    function collateral(address _agent) public view returns (address collateralAddress) {
```

Comment:
Does this need to be public? I don't see it used by this contract.

Reply:
Agreed, it was in a previous version of the PR but now it's obsolete. Made the change in the next commit